packages/web-react/src/components/Breadcrumbs/README.md

```diff
@@ -0,0 +1,38 @@
+# Breadcrumbs
+
+```jsx
+import { Breadcrumbs } from '@lmc-eu/spirit-web-react/components';
+```
+
+```jsx
+const items = [
+  {
+    title: 'Root',
+    url: '#',
+  },
+  {
+    title: 'Category',
+    url: '#',
+  },
+  {
+    title: 'Subcategory',
+    url: '#',
+  },
+  {
+    title: 'Current page',
+    url: '#',
+  },
+];
+```
+
+```jsx
+<Breadcrumbs items={items} />
+```
+
+## Available props
+
+| Name    | Type                           | Description           |
+| ------- | ------------------------------ | --------------------- |
+| `items` | `{ title: 'Title', url: '#' }` | Navigation menu items |
```

Comment:
There is a missing documented prop for a go-back item.

Reply:
README was updated.